common/views/components/Buttons/Control/Control.tsx

```diff
@@ -91,14 +206,14 @@ const Control = forwardRef(
             replace={replace}
             prefetch={prefetch}
           >
-            <a ref={ref} {...attrs}>
+            <Wrapper ref={ref} {...attrs}>
```

Comment:
we never want this to be an anchor anymore?

Reply:
good spot – this is supposed to have `as="a"` on it. Will update.